iam/policy-templates/dss-checkout-sfn-lambda.json

```diff
@@ -35,7 +35,9 @@
       ],
       "Resource": [
         "arn:aws:s3:::$DSS_S3_CHECKOUT_BUCKET",
-        "arn:aws:s3:::$DSS_S3_CHECKOUT_BUCKET/*"
+        "arn:aws:s3:::$DSS_S3_CHECKOUT_BUCKET/*",
+        "arn:aws:s3:::$DSS_S3_CHECKOUT_BUCKET_TEST",
```

Comment:
this is not specified anywhere.

Also, doesn't this need to be in `build_deploy_config.sh`?

Reply:
@ttung we discussed this comment and you agreed this was reasonable

Reply:
I think it's reasonable that you need this, but I think you still need to export the variable, otherwise it'll be undefined.  The iam policy templater does not have access to all environment variables.

Reply:
fixed, please see at the top.